Annotato/Annotato/Utils/NetworkMonitor.swift

```diff
@@ -9,7 +9,7 @@ class NetworkMonitor: ObservableObject {
     private let queue = DispatchQueue.global()
     private let monitor = NWPathMonitor()
 
-    func start() {
+    func start(webSocketManager: WebSocketManager?) {
```

Comment:
Let's not have the `NetworkMonitor` depend on `WebSocketManager`? We can do so by making the websocketmanager subscribe to `isConnected`